libsolidity/ast/Types.cpp

```diff
@@ -561,6 +812,75 @@ shared_ptr<IntegerType const> IntegerConstantType::integerType() const
 		);
 }
 
+shared_ptr<FixedPointType const> RationalNumberType::fixedPointType() const
+{
+	//do calculations up here
```

Comment:
also, getting an error thrown at me with this:

rational maxValue = rational(bigint(1) << 256);